_data/buildx/docker_buildx_build.yaml

```diff
@@ -588,6 +652,27 @@ examples: |-
   $ docker buildx create --use --name insecure-builder --buildkitd-flags '--allow-insecure-entitlement security.insecure'
   $ docker buildx build --allow security.insecure .
   ```
+
+  ### Size of `/dev/shm` (--shm-size) {#shm-size}
```

Comment:
We haven't shipped buildx 0.7 yet with Docker Desktop (or the linux packages), so perhaps we should wait for the next Docker Desktop release 🤔

Reply:
Which reminds me; could you open a PR to update docker-ce-packaging? https://github.com/docker/docker-ce-packaging/blob/master/plugins/buildx.installer

Reply:
Yes indeed we can keep it in draft in the meantime